### ISSUE TITLE
Using RTMP stream in a video-on-demand form

### ISSUE THREAD

**Issue by SuiriuS** [calzoneman/sync#692]
### Description of the Problem ###

RTMP live streaming seems to work well. I have an nginx server setup to proxy the stream and using a livestream in the format   rtmp://server.address:1935/live/test   is great, but I also have the nginx server setup with a simple video-on-demand where the file is stored locally and streamed over rtmp through   rtmp://server.address:1935/vod/nameofvideo.mp4  which returns a format not supported error on cytube.

I have confirmed that the vod stream works with vlc and with my own website running videojs player. And I have tried a .mp4 and .flv formats.

So in summary [ /live/test ] works, [ /vod/video.mp4 ] does not work.

### System Information ###

Using Chrome on Windows 10
Error is: Media could not be loaded either because the server or network failed or because the format is not supported.

### PS ###
Where do I submit feature suggestions/requests?

**Comment by calzoneman** [comment]
>Where do I submit feature suggestions/requests?
GitHub is fine.

CyTube already supports playing MP4 files over HTTPS, and Flash is dying a slow and painful death (taking RTMP with it), so I don't see the value in supporting this.  Is there any reason I'm missing that it would be better to support RTMP VOD rather than having the user serve the VOD over HTTPS instead?

**Comment by SuiriuS** [comment]
Alright, I'm new to all this and haven't looked into HTTPS yet, but I just solved the issue myself, as I suspected it is because of the file extensions and simply removing the extension from the local file and thus from the address works. So no need to change anything I suppose.

**Comment by calzoneman** [comment]
If you're playing VODs, I highly recommend just setting up a static file server with nginx (you can use [certbot](https://certbot.eff.org/) to get a free SSL certificate) and doing away with RTMP.  Many browsers are disabling Flash by default, and Flash isn't even available on many devices.

Marking this issue as closed.
